_posts/0000-01-02-DoctorDuende.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "¡Bienvenido a la segunda diapositiva!"
+---
+Esto va a suponer el comienzo de una nueva era
+Usa la flecha de la izquierda para ir atrás!
```

Comment:
Quite cool this feature. It gives you the chance to comment several lines **at once**.

Reply:
It is a great feature for sure. Congrats!